trevorwinser/Infinite-Loopers
# Site integration

Completed #149 and #127 

Test code using the following steps:

1. Navigate to _server_ directory using `cd server`
2. Setup your virtual environment.
3. Run `pip3 install -r ../requirements.txt`
4. Run `flask run`

This should open up a flask app, and navigating to http://127.0.0.1:5000 should bring you to the login page. **This is normal.** This is because all users that aren't logged in are automatically brought to the login page.

You can then login using the credentials:

Username: Admin1
Password: Password1

For the admin user, and

Username:  User1
Password: Password1

For the normal user.

## What's New?

* All users will be brought to the user dashboard, and their username will be displayed on the top-left.
* Admin users can navigate to the admin dashboard using the menu on the right, and can navigate back using the **EXIT** button.
* Normal users do not get an option for navigating to the admin page.
